source/user-manual/reference/tools/keystore-tool.rst

```diff
@@ -0,0 +1,35 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Tool where sensitive configuration data, including user credentials for the indexer, can be securely stored. 
+
+.. _keystore-tool:
+
+keystore-tool
+=============
+
+The keystore-tool increases the security of sensitive information, in this case it stores the username and password used by the Wazuh manager to connect to the indexer.
+
+With this tool, you can set these values generated by the indexer automatically.
```

Comment:
Ditto

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit